README.md

```diff
@@ -59,9 +58,28 @@ The following configuration flags will be respected:
 
 - `:use_https` toggles the use of HTTPS. Defaults to `true`
 - `:source` a String or Array that specifies the imgix Source address. Should be in the form of `"assets.imgix.net"`.
-- `:secure_url_token` a optional secure URL token found in your dashboard (https://webapp.imgix.com) used for signing requests
-- `:hostnames_to_replace` an Array of hostnames to replace with the value(s) specified by `:source`. This is useful if you store full-qualified S3 URLs in your database, but want to serve images through imgix.
-- `shard_strategy`: Specify [domain sharding strategy](https://github.com/imgix/imgix-rb#domain-sharded-urls) with multiple sources. Acceptable values are `:cycle` and `:crc`. `:crc` is used by default.
+- `:secure_url_token` an optional secure URL token found in your dashboard (https://webapp.imgix.com) used for signing requests
```

Comment:
Can we fix this to be `https://dashboard.imgix.com`? The redirect works, but "dashboard" is canon.